Comments/ClientApp/src/components/Pagination/Pagination.scss

```diff
@@ -18,7 +21,7 @@
       background: none;
       border: 1px solid transparent;
       background: none;
-      color: $colour-link;
+      color: colours.$link !important;
```

Comment:
Just a note about `!important` - we'd usually run screaming from it, but found that moving to a modular SCSS structure did cause some specificity issues to pop up due to the flattened structure, so it got added to button text to stop parent styles overriding the button's own, if you see what I mean. In the end Ian and I decided that using `!important` was fine as a last resort as long as the thing we're applying styles to was the "end of the line", i.e. it didn't have any child elements that might inherit the important rule and therefore be impossible to override. So this is fine as long as you're happy that it satisfies that requirement!

Reply:
Yeah, makes sense. I gathered it must have been set to important for a good reason, but obviously then forced to add it myself. The buttons are end of the line anyway, so all good.